### DISCUSSION TITLE
Failed to load Java Agent

### DISCUSSION THREAD

**Discussion by DavMontas** [Ginger-Automation/Ginger#2907] [Q&A]
Error details: 'String Parsing Error/wrong value type b=2 Name of the payload Is Error Buffer Index is: 14.

I think the application im trying to automate ask for some credentials in order for the agent to have permission to automate but I don't know where can I pass these credentials.

**Comment by manas-droid** [comment]
Hi @DavMontas , Do you still face this issue?

**Reply by DavMontas** [reply]
No, I don't

**Reply by manas-droid** [reply]
ok, Closing this issue then. 
Thanks!
